addon/modifiers/container-query.ts

```diff
@@ -154,14 +158,14 @@ export default class ContainerQueryModifier<
 
     for (const [featureName, meetsFeature] of Object.entries(
       this.queryResults
-    )) {
+    ) as ObjectEntries<QueryResults<T>>) {
```

Comment:
We defined `this.queryResults` (i.e. an external source did not). As a result, we should be able to assert that `Object.entries(this.queryResults)` has the type of `ObjectEntries` instead of [`type-fest`'s broader type, `Entries`](https://github.com/sindresorhus/type-fest/blob/main/source/entries.d.ts#L57-L62).